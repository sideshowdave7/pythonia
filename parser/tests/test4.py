b = a[2] << 3
def c(d, e):
 return (3, 2, 1)

def g():
 return 4, 5

def h(a):
 try:
  if False:
   return True
  elif 5 == 3:
   return -2+9
 except IOError:
  print("WHY!?!" "bob");
 finally:
  global b

