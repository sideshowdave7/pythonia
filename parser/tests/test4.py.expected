(program
 (= (b) (shift (indexed a (subscript 2)) ("<<" 3)))
 (def (c d e) (suite (return (tuple 3 2 1))))
 (def (g) (suite (return 4 5)))
 (def
  (h a)
  (suite
   (try
    (suite
     (cond
      (False (suite (return True)))
      ((comparison 5 ("==" 3)) (suite (return (arith ("-" 2) ("+" 9)))))))
    (((except IOError) (suite (expr (indexed print (called "WHY!?!bob"))))))
    #f
    (suite (global b))))))

